packages/CalendarCarousel/index.tsx

```diff
@@ -202,6 +226,41 @@ function CalendarCarousel<T>({
       numPrevMonthDays--;
     }
 
+    const markedDays = [7, 17, 27];
+    const markedDayEvents = [
+      {
+        selectedEventDate: new Date(2020, 7, 7),
+        year: 2020,
+        month: 6,
+        day: 7,
```

Comment:
I think we don't need separate variables for `year`, `month`, and `day` when we have `selectedEventDate`.

Reply:
yup fixed this!